### DISCUSSION TITLE
is rook support to use btrfs for storageclass?

### DISCUSSION THREAD

**Discussion by panyang1217** [rook/rook#7468] [Q&A] [answered]
When I mount huge ext4 volume (about 1pib ), it take serval minutes when pod start. 
So I want to use btrfs for this huge volume, is rook support to using btrfs with storageclass?

**Comment by galexrt** [comment]
I have updated the title and labels, assuming @panyang1217 you are talking about Rook Ceph.

**Comment by leseb** [comment, marked as answer]
I think it's more a ceph-csi question, AFAIK it only supports `xfs` and `ext4` but I'll leave @Madhu-1 or @nixpanic to confirm.

**Reply by nixpanic** [reply]
That is correct, Ceph-CSI only supports `ext4` (default) and `xfs` at the moment.

The CephCSI project uses CentOS as base container images (just like the Ceph project). btrfs is not readily available there, so it is not a feature that can easily be added. It is of course possible to open an issue and propose a PR for adding support, go to https://github.com/ceph/ceph-csi for doing that.
